test/test_util/test_utilities.py

```diff
@@ -85,19 +87,19 @@ def test_save_results(self):
 
 class LoadResultsTestCase(unittest.TestCase):
     def test_load_results(self):
-        # test for 1d
-
-        # test for 3d
-
         nr_experiments = 10000
 
         # test for 2d
-        experiments = pd.DataFrame(
-            index=np.arange(nr_experiments), columns={"x": float, "y": float}
+        cases = np.empty(
+            nr_experiments, dtype=[("x", float), ("y", int), ("z", bool), ("q", object)]
         )
+        experiments = pd.DataFrame.from_records(cases)
 
         experiments["x"] = np.random.rand(nr_experiments)
-        experiments["y"] = np.random.rand(nr_experiments)
+        experiments["y"] = np.random.randint(0, 10, size=nr_experiments)
+        experiments["y"] = np.random.randint(0, 1, size=nr_experiments, dtype=bool)
```

Comment:
This line seems to overwrite the previous value of `experiments["y"]` in the line before (it's assigned twice, so the first is overwritten). Is that intended?

Reply:
should be "z". See extra commit